tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -15,23 +15,23 @@
 
 import random
 import time
-from ducktape.mark import ignore, matrix
+from ducktape.mark import ignore
+from ducktape.mark import matrix
 from ducktape.mark.resource import cluster
 from ducktape.tests.test import Test
 from kafkatest.services.kafka import KafkaService
 from kafkatest.services.streams import StreamsSmokeTestDriverService, StreamsSmokeTestJobRunnerService, StreamsUpgradeTestJobRunnerService
 from kafkatest.services.zookeeper import ZookeeperService
-from kafkatest.version import LATEST_0_10_0, LATEST_0_10_1, LATEST_0_10_2, LATEST_0_11_0, LATEST_1_0, LATEST_1_1, DEV_BRANCH, DEV_VERSION, KafkaVersion
+from kafkatest.version import LATEST_0_10_0, LATEST_0_10_1, LATEST_0_10_2, LATEST_0_11_0, LATEST_1_0, LATEST_1_1, LATEST_2_0, DEV_BRANCH, DEV_VERSION, KafkaVersion
 
 # broker 0.10.0 is not compatible with newer Kafka Streams versions
-broker_upgrade_versions = [str(LATEST_0_10_1), str(LATEST_0_10_2), str(LATEST_0_11_0), str(LATEST_1_0), str(LATEST_1_1), str(DEV_BRANCH)]
+broker_upgrade_versions = [str(LATEST_0_10_1), str(LATEST_0_10_2), str(LATEST_0_11_0), str(LATEST_1_0), str(LATEST_1_1), str(LATEST_2_0), str(DEV_BRANCH)]
 
 metadata_1_versions = [str(LATEST_0_10_0)]
 metadata_2_versions = [str(LATEST_0_10_1), str(LATEST_0_10_2), str(LATEST_0_11_0), str(LATEST_1_0), str(LATEST_1_1)]
-# we can add the following versions to `backward_compatible_metadata_2_versions` after the corresponding
-# bug-fix release 0.10.1.2, 0.10.2.2, 0.11.0.3, 1.0.2, and 1.1.1 are available:
-# str(LATEST_0_10_1), str(LATEST_0_10_2), str(LATEST_0_11_0), str(LATEST_1_0), str(LATEST_1_1)
-backward_compatible_metadata_2_versions = []
+# once 0.10.1.2 is available backward_compatible_metadata_2_versions
+# can be replaced with metadata_2_versions
+backward_compatible_metadata_2_versions = [str(LATEST_0_10_2), str(LATEST_0_11_0), str(LATEST_1_0), str(LATEST_1_1)]
 metadata_3_versions = [str(DEV_VERSION)]
```

Comment:
Should this be `[str(LATEST_2_0), str(DEV_VERSION)]` ?

Or actually, `trunk` is already on version 4 -- however, since version 3 we have "version probing" and thus we don't need to distinguish versions 3,4,... -- maybe we can rename the variable to `metadata_3_or_higher_versions` (or a similar but better name) ?

Reply:
ack for adding `LATEST_2_0`.  As for the name I can't come up with anything better at the moment so I'll go with what you suggested.